test/cmdlineTests.sh

```diff
@@ -306,6 +305,21 @@ SOLTMPDIR=$(mktemp -d)
     REPO_ROOT=$(pwd) # make it absolute
     cd "$SOLTMPDIR"
     "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/docs/ docs
+
+    if node -v >/dev/null
```

Comment:
```suggestion
    if node -v >/dev/null 2>&1
```

Reply:
Also, why do you check for node if what you actually want is npm?

Reply:
That's a good point, I was still testing this tbh as waiting for each PR to run the tests each time is taking a while, some of these changes were not really final, but this is a good suggestion 👍